runtime/CMakeLists.txt

```diff
@@ -150,6 +150,18 @@ else()
 	add_custom_target(j9vm_nlsgen)
 endif()
 
+# Generate jvmti.h
+if(J9VM_IS_NON_STAGING)
+	set(JVMTI_HEADER_DIR  "${CMAKE_CURRENT_BINARY_DIR}/include")
```

Comment:
@dnakamura suggests that adding
```
	file(MAKE_DIRECTORY "${JVMTI_HEADER_DIR}")
```
after this line should fix the problem we see in in the travis lint build:
```
/bin/sh: 1: cd: can't cd to /home/travis/build/eclipse/openj9/build/runtime/include
runtime/CMakeFiles/j9vm_m4gen.dir/build.make:73: recipe for target 'runtime/include/jvmti.h' failed
```

Reply:
already add it, hope this will fix it.